disnake/enums.py

```diff
@@ -1560,7 +508,7 @@ def category(self) -> Optional[AuditLogActionCategory]:
 
     @property
     def target_type(self) -> Optional[str]:
-        """The target of this :class:`AuditLogAction`."""
```

Comment:
.

Reply:
Resolved in 606392a3f3350c8f1e22fa43c4b268855841d5cf.